docs/extensions/toc.md

```diff
@@ -196,6 +196,14 @@ The following options are provided to configure the output:
 * **`separator`**:
     Word separator. Character which replaces white space in id. Defaults to "`-`".
 
+* **`toc_height`**
+    Define from to which section level "n" (`<hn>` to `<h6>`, where `1 <= n <= 6`)
+    to include in the Table of Contents. Defaults to `1`.
```

Comment:
There must be a typo here. "from to which" can' the correct. I'm not even sure what it is supposed to say.